web/packages/design/src/SlideTabs/SlideTabs.story.tsx

```diff
@@ -49,32 +72,90 @@ export const FiveTabs = () => {
 export const Round = () => {
   const [activeIndex, setActiveIndex] = useState(0);
   return (
-    <SlideTabs
-      appearance="round"
-      tabs={['step1', 'step2', 'step3', 'step4', 'step5']}
-      onChange={setActiveIndex}
-      activeIndex={activeIndex}
-    />
+    <Flex flexDirection="column" gap={3}>
+      <SlideTabs
+        appearance="round"
+        tabs={fiveSimpleTabs}
+        onChange={setActiveIndex}
+        activeIndex={activeIndex}
+      />
+      <SlideTabs
+        tabs={titlesWithIcons}
+        appearance="round"
+        onChange={setActiveIndex}
+        activeIndex={activeIndex}
+      />
+    </Flex>
   );
 };
 
 export const Medium = () => {
   const [activeIndex, setActiveIndex] = useState(0);
```

Comment:
Can we have a separate state for each slider? They have a different number of steps, so you can visually simulate an 'index out of range' error :)

https://github.com/user-attachments/assets/ba444239-e2c6-49b3-bf43-6d3bd434fac2